docs/WritingTests.asciidoc

```diff
@@ -672,8 +672,8 @@ eventually workers can be held back for the cluster.
 It is possible to ensure that all jobs within the same _parallel_ cluster are
 executed on the same worker host. This is useful for connecting the SUTs without
 having to connect the physical worker hosts. Use `PARALLEL_ONE_HOST_ONLY=1` to
-enable this. Note that adding this setting in `workers.ini` has currently *no*
-effect.
+enable this. It is also possible to enable this setting as a global in
+`workers.ini`.
```

Comment:
```suggestion
enable this. This setting can be applied as a test variable during the time
of scheduling as well as in the worker configuration file `workers.ini`.
```

Reply:
Done

Reply:
I don't see the suggestion by @okurz applied (but would also prefer his phrasing).

Reply:
yes, apparently I forgot to push. most of them should be there now